### PR TITLE
implement suggested changes to default text

### PR DESCRIPTION
**Pull request by moloko**
see #18 

It's maybe a little wordy on small screens?

pip

### CODE REVIEW

**Comment by chucklorenz** [comment]
A pat on the back for moving in this direction. Always nice to have good defaults as a jumping off point.

Agree with Matt that the above is a bit wordy. Here's a draft that might help editors along:

You haven’t looked at all the content on this page.
If you’d like to complete this page, select **Stay on the page**. Select the progress bar at the top of the screen to see what's left to do. Jump to what's incomplete by selecting a heading.
If you’d like to leave the page, select **Exit page**.

**Comment by moloko** [comment]
Updated screenshot:
image

**Comment by moloko** [comment]
come on @kineopete you get one PR to review and you ignore it for TWO YEARS?!
